pytest.ini

```diff
@@ -14,5 +14,6 @@ asyncio_mode = auto
 asyncio_default_fixture_loop_scope = function
 filterwarnings =
     error
+    ignore:'asyncio.iscoroutinefunction' is deprecated:DeprecationWarning:aiohttp
```

Comment:
Shouldn't we just fix this one?
Looks like it should be changed to inspect.iscoroutinefunction().

Reply:
Yes, we should fix it in aiohttp and temporarily ignore it for aiojobs.
I have a plan to enable py3.14 alpha for aiohttp and see how it is going.

Reply:
Yes, we should fix it in aiohttp and temporarily ignore it for aiojobs.
I have a plan to enable py3.14 alpha for aiohttp and see how it is going.